libcst/codemod/visitors/_gather_unused_imports.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+
+from typing import Dict, List, Optional, Sequence, Set, Tuple, Union, Iterable
+
+import libcst as cst
+import libcst.matchers as m
+from libcst.codemod._context import CodemodContext
+from libcst.codemod._visitor import ContextAwareVisitor
+from libcst.metadata import QualifiedNameProvider, ScopeProvider, PositionProvider
+from libcst.codemod.visitors._gather_exports import GatherExportsVisitor
+from libcst.codemod.visitors._gather_comments import GatherCommentsVisitor
+from libcst.codemod.visitors._gather_string_annotation_names import (
+    GatherNamesFromStringAnnotationsVisitor,
+)
+from libcst.metadata.scope_provider import _gen_dotted_names
+
+
+DEFAULT_SUPPRESS_COMMENT_REGEX = (
+    r".*\W(lint-ignore: ?unused-import|noqa|lint-ignore: ?F401)(\W.*)?$"
```

Comment:
I suggest this one instead:

```py
r".*\W(noqa[^:]|noqa: ?F401|lint-ignore: ?unused-import|lint-ignore: ?F401)(\W.*)?$"
```

So it ignores things like `# noqa`, `# noqa: F401`, but not `# noqa: IG61`